instantout/store_test.go

```diff
@@ -0,0 +1,36 @@
+package instantout
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/lightninglabs/loop/instantout/reservation"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConvertingReservations(t *testing.T) {
```

Comment:
I think some more coverage would be nice to have.

Reply:
Bumping this, but given the isolated nature if this feature, perhaps we can ease review load a bit by separating the tests to a follow up PR to enable testnet testing.